### PR TITLE
dApp: Updated UDC screen

### LINKED ISSUE #1768: Update UDC screen
## User Story
As Chris I would like to have a cleaner looking UDC screen. Split from issue #587 

## Acceptance criteria
- Display a short description of the UDC and utility token
- Display current **deposited** utility token amount
- Display icon for depositing utility tokens
- Display icon for withdrawing utility tokens
- Display short warning if the utility token amount is less than what is required for enabling monitoring

## Tasks
- [ ] Needs mockup
- [ ] In-iteration design

### PR DESCRIPTION
**Pull request by taleldayekh**
Fixes #1768 

**Short description**
This PR takes care of the UDC screen rework.
Screenshot 2020-07-21 at 16 04 48

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1. Connect to the dApp and click to the UDC screen in the Account menu.
2. Make a deposit.
3. Make a withdrawal.

### CODE REVIEW

File: raiden-dapp/src/views/account/UDC.vue [hunk 1]
```diff
@@ -1,133 +1,52 @@
...
-          )
-        "
-        class="udc__action"
-        @click="udcDeposit()"
-      />
+        <span class="udc__actions__button--title">
+          {{ $t('udc.deposit') }}
+        </span>
+      </v-btn>
+      <v-btn text class="udc__actions__button" @click="withdrawFromUdc = true">
```
Comment (weilbith) [inline raiden-dapp/src/views/account/UDC.vue:26]: Nitpick:
The button above has an action called `showUdcDeposit`. I would expect this do be similar. So I would either decide to say that we want to show a dialog or what the dialog its purpose is (e.g. deposit to udc).

File: raiden-dapp/src/views/account/UDC.vue [hunk 2]
```diff
@@ -1,133 +1,52 @@
...
-          v-model="amount"
-          autofocus
-          type="text"
-          :suffix="serviceToken"
-          class="udc__mint-deposit--amount"
+    <v-row class="udc__actions" no-gutters justify="center">
+      <v-btn text class="udc__actions__button" @click="showUdcDeposit = true">
+        <v-img
+          width="70px"
+          height="65px"
```
Comment (weilbith) [inline raiden-dapp/src/views/account/UDC.vue:19]: Nitpick^2:
I personally would prefer a smaller size. While the buttons appear fancy at the channel screen, here they look a bit chunky to me.

Comment (taleldayekh) [reply raiden-dapp/src/views/account/UDC.vue:19]: I made them slightly smaller. :)

File: raiden-dapp/src/views/account/UDC.vue (hunk 1 above)
Comment (taleldayekh) [reply raiden-dapp/src/views/account/UDC.vue:26]: I just kept the same variable name as in the other components where it is used, makes it a bit easier when searching I think.

**Pull request merged**